### PR TITLE
[frontend] Fix sorting of image templates

### PR DESCRIPTION
**Pull request by bgeuken**
* Shows templates from local projects first, and remotes afterwards.
  This broke with 89d7fc5.
* Sorts each template within a project by it's title. The title is what
  is visible by the user.

### CODE REVIEW

**Comment by bgeuken** [comment]
@adrianschroeter @ChrisBr Please review. This probably is still not perfect since we either show the title or the name of a template. Depending on what is available. But since the title is the default it makes more sense to sort by the title.

**Comment by bgeuken** [comment]
Codecov report can be found here: https://codecov.io/gh/openSUSE/open-build-service/pull/3974

File: src/api/app/models/project.rb [hunk 1]
```diff
@@ -109,7 +109,8 @@ def autocomplete(search)
   # will return all projects with attribute 'OBS:ImageTemplates'
   scope :local_image_templates, lambda {
     includes(:packages).joins(attribs: { attrib_type: :attrib_namespace }).
-      where(attrib_types: { name: 'ImageTemplates' }, attrib_namespaces: { name: 'OBS' })
+      where(attrib_types: { name: 'ImageTemplates' }, attrib_namespaces: { name: 'OBS' }).
+      order(:name)
```
Comment (ChrisBr) [inline src/api/app/models/project.rb]: by ``:title``

and then you could drop the view because it is already sorted, right?

**Comment by bgeuken** [comment]
@ChrisBr updated

File: src/api/app/views/webui/image_templates/index.xml.builder [hunk 2]
```diff
@@ -10,4 +10,4 @@ xml.image_template_projects do
       end
     end
   end
-end
\ No newline at end of file
+end
```
Comment (ChrisBr) [inline src/api/app/views/webui/image_templates/index.xml.builder]: [nitpick] doesn't belong to this PR
